core/src/main/java/com/redhat/lightblue/client/request/data/DataDeleteRequest.java

```diff
@@ -31,6 +31,7 @@ public JsonNode getBodyJson() {
         if (queryExpression != null) {
             node.set("query", queryExpression.toJson());
         }
+        appendExecutionToJson(node);
```

Comment:
Perhaps we should spawn this discussion into it's own issue? That said, we could look at doing something like what we already do in lightblue-core and use interfaces... see https://github.com/lightblue-platform/lightblue-core/blob/master/crud/src/main/java/com/redhat/lightblue/crud/WithRange.java